core/src/plugins/filed/python/vmware/bareos-fd-vmware.py

```diff
@@ -124,6 +124,7 @@ def __init__(self, plugindef):
             "vadp_dumper_sectors_per_call",
             "vadp_dumper_query_allocated_blocks_chunk_size",
             "fallback_to_full_cbt",
+            "restore_ignore_disks_mismatch",
```

Comment:
As there is still a M_WARNING when this happens, I'd argue we do not ignore it, but just allow it.

```suggestion
            "restore_allow_disks_mismatch",
```